app/api/__init__.py

```diff
@@ -398,3 +401,7 @@
 api.route(OrderRelationship, 'order_event', '/orders/<identifier>/relationships/event')
 api.route(OrderRelationship, 'order_marketer', '/orders/<identifier>/relationships/marketer')
 api.route(OrderRelationship, 'order_discount', '/orders/<identifier>/relationships/discount-code')
+
+# Event Analytics API
+api.route(EventAnalyticsGeneralDetail, 'event_analytics_general_detail', '/analytics/events/<int:id>/general',
```

Comment:
Because ... Something like `analytics/events/{event_identifier}/general` is pretty divergent from json-api spec.